src/components/BadgeClose.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styled from "styled-components";
+import CloseButton from "./CloseButton";
+
+const WrapperBadge = styled.div`
+  margin: 25px 10px;
+`;
+
+const BadgeSpan = styled.span`
+  border-radius: 5px;
+  padding: 4px 10px 3px 10px;
+  border: none;
+  box-shadow: 2px 2px 2px #00000029;
+  font-family: "Fira Sans", sans-serif;
+  font-weight: 900;
+  font-size: 21px;
+  background: #242623;
+  color: #ffffff;
+  cursor: pointer;
+`;
+
+const StyledCloseButton = styled.span`
+  position: absolute;
+  left: 114px;
+  top: -2px;
```

Comment:
You should use `right` instead of `left` here, because you want the badge to stick to the right side. Also `BadgeSpan` needs a `position: relative` for the `StyledCloseButton`to stick to it

Reply:
👍